src/pudl_archiver/archivers/eia860m.py

```diff
@@ -31,13 +31,13 @@ async def get_resources(self) -> ArchiveAwaitable:
     async def get_year_month_resource(
         self, link: str, match: typing.Match
     ) -> tuple[Path, dict]:
-        """Download zip file."""
+        """Download xlsx file."""
         url = f"https://eia.gov/{link}"
         year = match.group(2)
         month = self.month_map[match.group(1)]
-        download_path = self.download_directory / f"eia860m-{year}-{month}.zip"
-        await self.download_zipfile(url, download_path)
+        download_path = self.download_directory / f"eia860m-{year}-{month:02}.xlsx"
```

Comment:
Was naming files incorrectly, and setting parts wrong.

Reply:
Nice find :)